src/test/java/wolox/training/services/UtilsForTest/OpenLibraryResponse.java

```diff
@@ -0,0 +1,83 @@
+package wolox.training.services.UtilsForTest;
+
+import com.google.common.base.Strings;
+import java.util.Map;
+
+public class OpenLibraryResponse {
+    public String whenItFoundsTheBook(Map bookState){
+        String isbnString= Strings.padStart(bookState.get("Isbn").toString(),10,'0');
+
+        String string=  "{"
```

Comment:
Check this variable name, you could name it like "correctResponse" or something more specific than string 😁

Reply:
Done :D